Utilities/Mergers/include/Mergers/ObjectStore.h

```diff
@@ -29,17 +31,27 @@ namespace o2::mergers
 class MergeInterface;
 
 using TObjectPtr = std::shared_ptr<TObject>;
+using VectorOfTObject = std::vector<TObject*>;
+using VectorOfTObjectPtr = std::vector<TObjectPtr>;
 using MergeInterfacePtr = std::shared_ptr<MergeInterface>;
```

Comment:
```suggestion
using TObjectShrPtr = std::shared_ptr<TObject>;
using VectorOfTObjectPtr = std::vector<TObject*>;
using VectorOfTObjectShrPtr = std::vector<TObjectPtr>;
using MergeInterfaceShrPtr = std::shared_ptr<MergeInterface>;
```
I think it is confusing later in the code that `VectorOfTObject` stores pointers while `VectorOfTObjectPtr` stores shared pointers. Perhaps we could make it more explicit.